ci/gpu/build.sh

```diff
@@ -30,7 +30,7 @@ nvidia-smi
 
 logger "Activate conda env..."
 source activate gdf
-conda install -c rapidsai/label/cuda${CUDA_REL} -c rapidsai-nightly/label/cuda${CUDA_REL} nvstrings=0.3*
+conda install -c rapidsai/label/cuda${CUDA_REL} -c rapidsai-nightly/label/cuda${CUDA_REL} nvstrings=0.7.*
```

Comment:
Might be best to redefine the nvstrings version as an environment variable and just put it at the top of the script.